index.md

```diff
@@ -33,7 +34,7 @@ Here's a simple program:
 
 ```val
 subscript longer_of(_ a: inout String, _ b: inout String): String {
-  yield if b.count() > a.count() { &b } else { &a }
+  inout { if b.count() > a.count() { &b } else { &a } }
```

Comment:
Hmm, conflicts with my change.  But OK, we don't need to yield if it's a single expression?

Reply:
Yep, `yield` and `return` are consistently implied for expression-bodied subscripts and functions